_docs_explore/61-data-model.md

```diff
@@ -273,4 +273,124 @@ The Local Attribute Share Info help to keep track of how the Local Attribute was
+| items     | (`RequestItemGroup` \| `RequestItem`)[] | A list of Request Items and Groups that are part of the Request. There must be at least one Item or Group per Request.                                                                                                                                                                |
+| metadata  | `string` \| `undefined`                 | Optional custom metadata that can be sent together with the Request. This property is meant purely for developers who integrate with Enmeshed. They can write for example some kind of key into this property, which can be used later to identify the content of this Request.       |         |     |     |
+
+### RequestItem
+
+Request Items can be sent inside of a Request and specify what should be done when the Request is accepted. `RequestItem` itself only defines some common properties. There are multiple types that inherit from `RequestItem`, like `CreateAttributeRequestItem` or `ReadAttributeRequestItem`.
+
+The base properties are:
+
+| Name           | Type                    | Description                                                                                                                                                                                                                                         | Remarks |
+| -------------- | ----------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| title          | `string` \| `undefined` | An optional, human readable title for the Request Item.                                                                                                                                                                                             |         |
+| description    | `string` \| `undefined` | An optional, human readable description for the Request Item.                                                                                                                                                                                       |         |
+| metadata       | `object` \| `undefined` | Optional metadata that can be sent together with this Request Item. The intended usage is the same as of the metadata property of the Request.                                                                                                      |         |
+| mustBeAccepted | boolean                 | If set to `true`, then this Request Item has to be accepted when the Request is accepted. Note that if the Request Item is inside of a Request Item Group, and the Request Item Group is rejected, then this Request Item must **not** be accepted. |         |
```

Comment:
requiredManualDecision fehlt

Reply:
Gut aufgepasst. Man merkt, dass ich den Part schon vor einigen Monaten geschrieben habe. Ist sonst noch was dazu gekommen in letzter Zeit?